src/sentry/relay/config/__init__.py

```diff
@@ -242,6 +242,46 @@ def _get_tx_name_rule(pattern: str, seen_last: int) -> TransactionNameRule:
     )
 
 
+class SpanDescriptionScope(TypedDict):
+    op: Literal["http"]
+    """Top scope to match on. Subscopes match all top scopes; for example, the
+    scope `http` matches `http.client` and `http.server` operations."""
+
+
+class SpanDescriptionRuleRedaction(TypedDict):
+    method: Literal["replace"]
+    substitution: str
```

Comment:
nit: We could use a single `RuleRedaction` for `txNameRules` and `spanDescriptionRules`.